src/core/file_format/trp.h

```diff
@@ -0,0 +1,45 @@
+// SPDX-FileCopyrightText: Copyright 2024 shadPS4 Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <vector>
+#include "common/endian.h"
+#include "core/crypto/crypto.h"
+#include "src/common/io_file.h"
+#include "src/common/types.h"
+
+struct trp_header {
```

Comment:
```suggestion
struct TrpHeader {
```
Also ditto for below struct